File: mario.c
// less

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);
    for (int i = 0; i < n; i++)
    {
        // draw the line for the first shape
        for (int j = 0; j < n - i - 1; j++) // draw the line regarding to the height-number of #
        {
            printf(" ");
        }
        for (int z = 0; z < i + 1; z++)
        {
            printf("#");
        }

        printf("  ");  // print space between the 2 triangles

        // draw the line for the second shape
        for (int z = 0; z < i + 1; z++)
        {
            printf("#");
        }

        printf("\n");  // to go to the next line
    }
}

File: mario.py
# to use get int
from cs50 import get_int

# asking for the height until get a number between 1 and 8
while True:
    n = get_int("Height: ")
    if n >= 1 and n <= 8:
        break

# draw the lines
for i in range(n):
    for j in range(n-i-1):  # draws the spaces on the left side
        print(" ", end="")
    for z in range(i+1):   # draws the hashes on the left side
        print("#", end="")

    print(end="  ")                 # print space between the 2 triangles

    for z in range(i+1):  # draw the line for the second shape
        print("#", end="")

    print()  # to go to the next line

